### PR TITLE
Added meters, inHg, and hPa to !convert

### PR DESCRIPTION
**Pull request by inkdracula**
Adds meters, pascal and inches of mercury as conversion units to !convert.

### CODE REVIEW

File: sintia/sintia.py [hunk 1]
```diff
@@ -505,20 +505,30 @@ def format_quote(self, quote: Quote) -> str:
             'km': 'kilometers',
             'lbs': 'pounds',
             'kg': 'kilograms',
+            'g': 'grams',
+            'm': 'meters',
+            'hPa': 'hectopascal',
+            'inHg': 'inch of mercury',
+            'inHg': 'inches of mercury',
+            'inHg': 'inhg'
+            'hPa': 'hpa'
```
Comment (LiraNuna) [inline sintia/sintia.py]: missing commas

File: sintia/sintia.py [hunk 2]
```diff
@@ -505,20 +505,30 @@ def format_quote(self, quote: Quote) -> str:
...
         }
 
         conversions = {
             'feet': {
                 'centimeter': lambda ft: ft * 30.48,
                 'inch': lambda inch: inch * 12.0,
+                'meters': lambda meters / 3.2808,
```
Comment (LiraNuna) [inline sintia/sintia.py]: missing `:`

File: sintia/sintia.py (hunk 1 above)
Comment (LiraNuna) [inline sintia/sintia.py]: You can also add pascal as an alias

**Review by LiraNuna — CHANGES_REQUESTED** [review]

**Comment by inkdracula** [comment]
added pascals to aliases

File: sintia/sintia.py [hunk 3]
```diff
@@ -506,20 +506,30 @@ def format_quote(self, quote: Quote) -> str:
             'lbs': 'pounds',
             'kg': 'kilograms',
             'g': 'grams',
+            'm': 'meters',
+            'hPa': 'hectopascal',
+            'inHg': 'inch of mercury',
+            'inHg': 'inches of mercury',
+            'inHg': 'inhg',
+            'hPa': 'hpa',
+            'hPa': 'pascals',
```
Comment (LiraNuna) [inline sintia/sintia.py]: The mapping here is reversed. it's 

what is the alias: what it maps to

you know it's right if the unit on the right is what is in the conversion map on the bottom.

**Review by LiraNuna — APPROVED** [review]

**Pull request merged**